arch/riscv/core/irq_manage.c

```diff
@@ -50,6 +50,7 @@ int arch_irq_connect_dynamic(unsigned int irq, unsigned int priority,
 	}
 #else
 	ARG_UNUSED(priority);
+	ARG_UNUSED(level);
```

Comment:
@galak why not defining level inside the #if ?

Reply:
Because I don't know what version of C we are conforming to and if we allow defining variables at any point.

Reply:
I think we've generally agreed on being no more than 20 years behind standard revisions, so C99 is assumed.

Reply:
Ok, re-worked the code to remove `level` completely.